### PR TITLE
Add new projects and code.

### PR DESCRIPTION
**Pull request by mikem8361**
Remove the old EmbedIndex, DownloadDumpFiles and Microsoft.SymbolStore.Client projects.

Add new Microsoft.FileFormats, Microsoft.SymbolStore and dotnet-symbols projects.

### CODE REVIEW

File: src/Directory.Build.props [hunk 1]
```diff
@@ -5,17 +5,13 @@
...
   <Import Project="$(RepoToolsetDir)Settings.props" />
 
-  <PropertyGroup>
-    <PackageIconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</PackageIconUrl>
-  </PropertyGroup>
-
   <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
     <CheckForOverflowUnderflow>false</CheckForOverflowUnderflow>
-    <DebugType>portable</DebugType>
+    <DebugType>full</DebugType>
```
Comment (tmat) [inline src/Directory.Build.props]: >full [](start = 15, length = 4)

Any reason to use Windows PDBs?

File: src/dotnet-symbols/dotnet-symbols.csproj [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <NoWarn>;1591;1701</NoWarn>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>dotnet-symbols</ToolCommandName>
+    <Description>Symbols download utility</Description>
+    <PackageTags>Symbols</PackageTags>
+    <PackageReleaseNotes>$(Description)</PackageReleaseNotes>
+    <RootNamespace>dotnet.symbols</RootNamespace>
```
Comment (tmat) [inline src/dotnet-symbols/dotnet-symbols.csproj]: >d [](start = 19, length = 1)

DotNet.Symbols?

**Comment by mikem8361** [comment]
This is the beginning of a dotnet cli global tool. So to get a command like “dotnet symbols” it needs to be lower case. Does that address your comment?

**Comment by mikem8361** [comment]
This code has to run on the desktop (net45) and netstandard 2.0 so I’ll try to make it condition.

**Pull request merged**